src/pages/responsible-trading/index.js

```diff
@@ -33,10 +41,40 @@ const StyledHeader = styled(Header)`
     }
 `
 
+const CFDWrapper = styled(Text)`
+    background-color: #f9fafc;
+    background-size: cover;
+    height: auto;
+    padding: 2rem 8rem;
+    max-height: 14rem;
+    overflow-y: auto;
+
+    @media ${device.tablet} {
+        padding: 1rem 4rem;
+    }
+
+    @media ${device.mobileL} {
+        padding: 1rem 2rem;
+    }
+`
+
+const CFDWarning = () => {
+    return (
+        <CFDWrapper>
+            {localize(
+                `CFDs are complex instruments and come with a high risk of losing money rapidly due to leverage. 74% of retail investor accounts lose money when trading CFDs with this provider. You should consider whether you understand how CFDs work and whether you can afford to take the high risk of losing your money.`,
+            )}
+        </CFDWrapper>
+    )
+}
+
 const ResponsibleTrading = () => {
-    const HeroBackground = isEuCountry ? EuBackground : NoneEuBackground
+    const { is_eu_country } = React.useContext(DerivStore)
+    const Hero = is_eu_country ? HeroEu : HeroNonEu
+    const HeroBackground = is_eu_country ? EuBackground : NoneEuBackground
+
     return (
-        <Layout>
+        <Layout CompotentAbove={is_eu_country ? CFDWarning : undefined}>
```

Comment:
```suggestion
        <Layout CompotentAbove={is_eu_country && CFDWarning}>
```

Reply:
done.

Reply:
changed proptypes to also accept bool for avoiding type mismatch error.